app/javascript/components/upload/ClusteringStep.js

```diff
@@ -8,7 +8,7 @@ const DEFAULT_NEW_CLUSTER_FILE = {
   file_type: 'Cluster'
 }
 
-export const clusterFileFilter = file => file.file_type === 'Cluster'
+export const clusterFileFilter = file => file.file_type === 'Cluster' && !file.is_spatial
```

Comment:
does this need a check on file existance ie `file?.file_type`

Reply:
I think the assumption is that the file filter will only be used on arrays of objects, so we don't need to check for existence